htdocs/langs/en_US/ftp.lang

```diff
@@ -0,0 +1,15 @@
+# FTP
```

Comment:
For several reason (performance purpose, disk optimization translation admin), we create a new language file only if we think we will rich 1000 lines. I dont think module ftp need a dedicated file multiplicated n times pers language dir for just few entries.
Please keep them into "other" that is dedicated for language files that are too small to justify its own language files

Reply:
OK i understand i change

@eldy it's done